source/command-monitoring/tests/unified/find.yml

```diff
@@ -0,0 +1,290 @@
+    operations:
+      - name: find
+        object: *collection
+        arguments:
+          filter: { _id: { $gt: 1 } }
+          sort: { _id: 1 }
+          skip: 2
+          comment: "test"
+          hint: { _id: 1 }
+          max: { _id: 6 }
+          maxTimeMS: 6000
+          min: { _id: 0 }
+          returnKey: false
+          showRecordId: false
+          readPreference:
+            mode: primaryPreferred
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                find: *collectionName
+                filter: { _id: { $gt: 1 } }
+                sort: { _id: 1 }
+                skip: 2
+                comment: "test"
+                hint: { _id: 1 }
+                max: { _id: 6 }
+                maxTimeMS: 6000
+                min: { _id: 0 }
+                returnKey: false
+                showRecordId: false
+              commandName: find
+              databaseName: *databaseName
+          - commandSucceededEvent:
+              reply:
+                ok: 1
+                cursor:
+                  id: 0
+                  ns: *namespace
+                  firstBatch:
+                    - { _id: 4, x: 44 }
+                    - { _id: 5, x: 55 }
+              commandName: find
+
+  - description: "A successful find with a getMore"
+    operations:
+      - name: find
+        object: *collection
+        arguments:
+          filter: { _id: { $gte: 1 }}
+          sort: { _id: 1 }
+          batchSize: 3
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                find: *collectionName
+                filter: { _id: { $gte: 1 }}
+                sort: { _id: 1 }
+                batchSize: 3
+              commandName: find
+              databaseName: *databaseName
+          - commandSucceededEvent:
+              reply:
+                ok: 1
+                cursor:
+                  id: { $$type: [ int, long ] }
+                  ns: *namespace
+                  firstBatch:
+                    - { _id: 1, x: 11 }
+                    - { _id: 2, x: 22 }
+                    - { _id: 3, x: 33 }
+              commandName: find
+          - commandStartedEvent:
+              command:
+                getMore: { $$type: [ int, long ] }
+                collection: *collectionName
+                batchSize: 3
+              commandName: getMore
+              databaseName: *databaseName
+          - commandSucceededEvent:
+              reply:
+                ok: 1
+                cursor:
+                  id: 0
+                  ns: *namespace
+                  nextBatch:
+                    - { _id: 4, x: 44 }
+                    - { _id: 5, x: 55 }
+              commandName: getMore
+
+  - description: "A successful find with a getMore and killCursors"
+    runOnRequirements:
+      - maxServerVersion: "2.6.99"
```

Comment:
we can probably just delete this test, unless anyone is still maintaining a driver version that supports 2.6? 

if they are, they should test this, as I can't with Swift or Rust.

Reply:
You can ask in `#driver-devs` but I think there's a few drivers that still support older servers. I'm not sure if they care about this spec test, though, as they might have their own tests in place for that behavior.